products/windows.md

```diff
@@ -63,6 +63,16 @@ releases:
     latest: 10.0.22631
     link: https://learn.microsoft.com/windows/release-health/windows11-release-information
 
+    # This cycle includes 10 22H2 (E) and commercial Pro from (W)
+-   releaseCycle: "10-22h2-esu"
+    releaseLabel: "10 22H2 ESU"
+    releaseDate: 2022-10-18
+    eoas: 2025-10-14
```

Comment:
Not sure if we should use `2025-10-14` or `2025-11-11` as `releaseDate`. I went the way that `10-21h2-iot-lts` did with having the regular release date and use `eoas` as it felt closer to the rest and more true to how using 22H2 will work. Also not sure if `lts: true` should be set as it's behaving like a LTS but officially something else.

Reply:
1. A little confused, since releaseDate is currently set to 2022. 
2. Should this be a draft PR since Windows 10 ESU isn't live yet?

Another option is to set `lts` to `2025-11-11` for the last 22H2 (Enterprise, Education, and Pro) release, and use `eoes`, which we use for Extended Security Updates.

Reply:
Fine for me as well with extended support. I just tried to keep it in sync with already existing variants for this OS (like IoT LTS)